exporter/clickhouseexporter/internal/gauge_metrics.go

```diff
@@ -122,16 +123,17 @@ func (g *gaugeMetrics) insert(ctx context.Context, db *sql.DB) error {
 			valueArgs[index+8] = model.metricDescription
 			valueArgs[index+9] = model.metricUnit
 			valueArgs[index+10] = attributesToMap(dp.Attributes())
-			valueArgs[index+11] = dp.Timestamp().AsTime().UnixNano()
-			valueArgs[index+12] = getValue(dp.IntValue(), dp.DoubleValue(), dp.ValueType())
-			valueArgs[index+13] = uint32(dp.Flags())
+			valueArgs[index+11] = dp.StartTimestamp().AsTime().UnixNano()
```

Comment:
Note: In Gauge, I experienced that the date was incorrect. According to the spec https://opentelemetry.io/docs/reference/specification/metrics/data-model/#gauge the field is optional. Probably, it is not supported.

Reply:
It's indeed still unclear what the start time for gauges is supposed to represent, and usually it's empty.

Reply:
I found that all of metric type's `start_time_unix_nano`  is optinal, see https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto